### DISCUSSION TITLE
Can we Update the Plugins Website?

### DISCUSSION THREAD

**Discussion by aljawaid** [kanboard/kanboard#5185] [General]
It is not clear who manages the forum so I will ask here...

@fguillot Can we change the html file for the plugins directory?  I'd like to sort it alphabetically but I dont want to spend all the time doing it if you dont accept the PR.

If you agree, we could also add a CSS file and style it better.

Over time, we could also add a new field for plugin developers to add a date of last update so new users and frequent users can identify which updates are more active than others.  This will improve Kanboard engagement with the community.

Kindly share your thoughts, decisions.

**Comment by fguillot** [comment]
> @fguillot Can we change the html file for the plugins directory? I'd like to sort it alphabetically but I dont want to spend all the time doing it if you dont accept the PR.

Sure.

> Over time, we could also add a new field for plugin developers to add a date of last update so new users and frequent users can identify which updates are more active than others. This will improve Kanboard engagement with the community.

It could be tedious to keep up to date, unless the process is automated somehow.

**Comment by aljawaid** [comment]
Thanks, I will work on it over time.... I think the plugins directory section needs a strong looking into.
